source/allocator.c

```diff
@@ -347,9 +329,7 @@ CFAllocatorRef aws_wrapped_cf_allocator_new(struct aws_allocator *allocator) {
 
     cf_allocator = CFAllocatorCreate(NULL, &context);
 
-    if (!cf_allocator) {
-        aws_raise_error(AWS_ERROR_OOM);
-    }
+    AWS_FATAL_ASSERT(cf_allocator && "creation of cf allocator failed!");
```

Comment:
```suggestion
    AWS_FATAL_ASSERT(cf_allocator, "creation of cf allocator failed!");
```

Reply:
the assert macro isn't done that way.

Reply:
Oh, you're right! So the string here is useless, right?

Reply:
it will be reported as part of the failed condition. This is how we've attached context to assert failures for 40 years :)